README.md

```diff
@@ -1,2 +1,10 @@
 # hidden-role-to-go-reactor
 Reactor for Hidden Role To Go Games
+
+## Rabbit
+https://spring.io/guides/gs/messaging-rabbitmq/
+
+brew link rabbitmq
+
+start `rabbitmq-server`
+`docker-compose up`
```

Comment:
is the docker startup required for rabbit? or should it get it's own section?

Reply:
Will redo the entire readme